src/GlobalTypes.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    toggleNav:'TOGGLE_NAV',
+    toggleInfo:'TOGGLE_INFO',
+    toggleSidebar:'TOGGLE_SIDEBAR',
+    handleNotesChange:'HANDLE_NOTES_CHANGE',
+    addNote:'ADD_NOTE',
+    removeNote:'REMOVE_NOTE',
+    restoreNote:'RESTORE_NOTE',
+    deleteNote:'DELETE_NOTE',
+    addTag:'ADD_TAG',
+    removeTag:'REMOVE_TAG',
+    changeQuery:'CHANGE_QUERY',
+    changeCurrentNote:'CHANGE_NOTE',
+};
```

Comment:
Thoughts on using `enum` instead? Kinda like:

```
export enum ActionTypes {
    TOGGLE_NAV = 'toggle-nav';
    TOGGLE_INFO= 'toggle-info';
}
```

You can use it like this:
```
case ActionType.TOGGLE_NAV:
```

Also, reason why the variable name is all caps like `TOGGLE_NAV` for the ActionType cause it is a constant. Kinda like a convention wherein constants (a non-changing value like PI = 3.14) should be in all caps.